packages/dom-membrane/tsconfig.rollup.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "../../tsconfig.json"
```

Comment:
I'm still fine-tuning tsconfig files, maybe we just need two: one for types and one for rollup